docs/Internal language.md

```diff
@@ -0,0 +1,79 @@
+# The model language
+
+It is important to understand the domain-specific language that describes the data models. It is what the input processors converts their inputs to, and the generators use to generate the output.
+
+It is also what developers can use to create their own models, since they can provide a **raw data model** as input. This way you can create your own custom data models that can be interpreted to any output language, with the full sweep of features the generators and presets support. See [Create your own models from the ground up, instead of a supported input](./advanced#create-your-own-models-from-the-ground-up-instead-of-a-supported-input)
```

Comment:
It is very hard to me to read this whole sentence. I feel I need an earlier answer for: What is `raw data model` ?

Reply:
Please see https://github.com/asyncapi/modelina/pull/530#issuecomment-1042000476